.buildkite/custom-tests.json

```diff
@@ -24,6 +24,21 @@
       "test_name": "fuzz-build",
       "command": "cd fuzz && cargo +nightly build",
       "platform": ["x86_64", "aarch64"]
+    },
+    {
+      "test_name": "fuzz-virtio-queue",
+      "command": "cd fuzz && cargo test --package common --lib virtio_queue && cargo +nightly fuzz run virtio_queue -- -max_total_time=240",
```

Comment:
The the default timeout from cargo fuzz is 1200s (inherited from `libfuzzer`), and is too big to report any related issues. Making the timeout `60s` or so should be more suitable.

Reply:
Hmmm, I do not completely understand why would we make the timeout less. My goal is to run for 15 minutes because that's not delaying the merge of PRs too much and also reaches a pretty good coverage in the given amount of time. Why are you suggesting the 60s?

Reply:
@andreeaflorescu The value of `-max_total_time` (15 mins) looks fine to me. I was talking about reducing the default value of `-timeout`, which limits the amount of time for a single run before it considers that run a failure/hanging [1]. The default value (1200s) is just too big to report meaningful hanging issues. A timeout of `60s` is used by the `oss-fuzz` project. I hope it clarifies the confusions.

[1] https://llvm.org/docs/LibFuzzer.html#options

Reply:
Ohhh, I see what you mean. That's super interesting. I'll make this update in a future PR if that's ok with you.